packages/utilities/gel-foundations/tests/getTypeSizes.test.js

```diff
@@ -1,25 +1,30 @@
-import { MEDIA_QUERY_TYPOGRAPHY } from '../src/breakpoints';
 import getTypeSizes from '../src/getTypeSizes';
-import { arabic } from '../src/scripts';
+import * as scripts from '../src/scripts';
 
 describe('getTypeSizes', () => {
-  it('Should return the Canon Type Sizes for the arabic script', () => {
-    const canonStyles = getTypeSizes('canon', arabic);
-    const block = `
-    font-size: 2rem;
-    line-height: 2.625rem;
+  Object.keys(scripts).forEach(scriptName => {
+    it.each`
+      typography
+      ${'atlas'}
+      ${'bodyCopy'}
+      ${'brevier'}
+      ${'canon'}
+      ${'doublePica'}
+      ${'elephant'}
+      ${'foolscap'}
+      ${'greatPrimer'}
+      ${'imperial'}
+      ${'longPrimer'}
+      ${'minion'}
+      ${'paragon'}
+      ${'pica'}
+      ${'royal'}
+      ${'trafalgar'}
+    `(`should match $typography for ${scriptName}`, ({ typography }) => {
+      const script = scripts[scriptName];
+      const styles = getTypeSizes(typography, script);
 
-    ${MEDIA_QUERY_TYPOGRAPHY.SMART_PHONE_ONLY} {
-      font-size: 2.375rem;
-      line-height: 2.875rem;
-    }
-
-    ${MEDIA_QUERY_TYPOGRAPHY.LAPTOP_AND_LARGER} {
-      font-size: 2.75rem;
-      line-height: 3.375rem;
-    }
-  `;
-
-    expect(block).toEqual(canonStyles);
+      expect(styles).toMatchSnapshot();
```

Comment:
I'm not really sure what these snapshot tests add here? They appear to show that font size and line height successfully translate into suitable `rem` values, could this be done with one test for `getTypeSizes` rather than one snapshot for each `typography` value?

Reply:
The idea behind the snapshot tests here was to also 'freeze' the current font-size and line-height values of all scripts. We weren't tracking changes to these and felt that was important to improve coverage here.

Reply:
Tbh, I kinda feel we can track changes like this through code reviews as it would need a review to change the values anyway. Happy to approved though :)